packages/dx-scheduler-core/src/plugins/view-state/reducers.js

```diff
@@ -4,9 +4,9 @@ export const changeCurrentDate = (currentDate, {
   nextDate,
   step,
   amount,
-  back = false,
+  direction = 'forward',
 }) => (
-  nextDate || moment(currentDate)[back ? 'subtract' : 'add'](amount, step).toDate()
+  nextDate || moment(currentDate)[direction === 'back' ? 'subtract' : 'add'](amount, step).toDate()
```

Comment:
In this case we can remove initial value for a direction argument.

`direction = 'forward', => direction,`